dist/index.html

```diff
@@ -13,7 +13,27 @@ <h1>Leaderboard</h1>
                 <span><h2>Recent scores <button type="button" id="refresh">Refresh</button></h2></span>
                 <table class="score-table">
                     <tbody class="board">
-                      
+                      <!-- <tr>
+                        <td>Name:100</td>
+                      </tr>
+                      <tr>
+                        <td>Name:20</td>
+                      </tr>
+                      <tr>
+                        <td>Name:50</td>
+                      </tr>
+                      <tr>
+                        <td>Name:78</td>
+                      </tr>
+                      <tr>
+                        <td>Name:125</td>
+                      </tr>
+                      <tr>
+                        <td>Name:77</td>
+                      </tr>
+                      <tr>
+                        <td>Name:42</td>
+                      </tr> -->
```

Comment:
- [ ] [Optional] Well done on converting all the functions into arrow functions! 👍 However, I noticed that you still have some commented code in your source code. Commented-out code is code that has been excluded or disabled from execution using special characters like "//" or "/* */".  Commented-out code takes up unnecessary space in your codebase and can make it harder to read and understand the active code. Commented-out code may cause confusion for other developers who are working on the project, as they might wonder if it's still relevant or intended to be used. 

- [ ] [Optional] To maintain a clean and organized codebase, I recommend removing all commented-out code that is no longer needed. If you need to keep a record of the old code for reference, consider using version control systems like Git to manage previous versions of your code.

- [ ] [Optional] By removing the commented-out code, you'll ensure that your codebase stays clear and concise, making it easier for you and your team to work on the project efficiently. Keep up the good work! 🚀🔧🧹